app/templates/curriculum-inventory-report/rollover.hbs

```diff
@@ -1,3 +1,12 @@
+<CurriculumInventory::ReportDetails
+  @report={{@model}}
+  @canUpdate={{this.canUpdate}}
+  @leadershipDetails={{this.leadershipDetails}}
+  @setLeadershipDetails={{this.setLeadershipDetails}}
+  @manageLeadership={{this.manageLeadership}}
+  @setManageLeadership={{this.setManageLeadership}}
+  @markReportAsFinalized={{set this.isFinalized true}}
```

Comment:
This is scary to me. I'd assume in the component that if I call `this.args.markReportAsFinalized` that it would be. But this just changes the controller state. Maybe a different name?

Reply:
:heavy_check_mark: